Sources/App/Models/Entities/AppUser/Extensions/AppUser+JWTKeychain.swift

```diff
@@ -14,7 +18,7 @@ extension AppUser: JWTCustomPayloadKeychainUserType {
     static let usernameKey: WritableKeyPath<AppUser, String> = \.email
     static let passwordKey: WritableKeyPath<AppUser, String> = \.password
 
-    struct UserLogin: HasReadablePassword, HasReadableUsername {
+    struct UserLogin: HasReadablePassword, HasReadableUsername, Decodable {
```

Comment:
Trying to keep the order the alphabetical ;)
```suggestion
    struct UserLogin: Decodable, HasReadablePassword, HasReadableUsername {
```